superset/assets/src/dashboard/util/getPublishSubscriberMap.js

```diff
@@ -131,14 +131,38 @@ export default function getPublishSubscriberMap(slices) {
 function updateSlices(slices) {
     let updatedSlices = _.clone(slices);
     updatedSlices.forEach(slice => {
-        let linkedSlices = getLinkedSlicesFromSubscriberLayer(slice.formData.subscriber_layers);
-        slice.formData.linked_slice = linkedSlices ? linkedSlices : slice.formData.linked_slice;
```

Comment:
we have added slice.formData.linked_slice for backward compatible of superset previous versions.  Why you removed that condition from line 135.

Reply:
updated